src/components/button/__next__/button.mdx

```diff
@@ -0,0 +1,169 @@
+| ------------- | ------- | --------- | -------- | ------ |
+| `default`     | ✅      | ✅        | ✅       | ✅     |
+| `destructive` | ✅      | ✅        | ❌       | ❌     |
+| `ai`          | ✅      | ❌        | ❌       | ❌     |
+
+By default, the `primary` variant type will be used if no `variantType` property
+is passed
+
+<Canvas of={ButtonStories.Variations} />
+
+### Sizes
+
+Buttons can be one of four sizes: `xs`, `small`, `medium` and `large`. Pass the
+required size with the `size` property.
+
+#### A note on the `xs` size
+
+The `xs` button size is designed for use only in tables where spacing is limited;
+**it is not meant for use beyond that**. XS buttons cannot use the `primary`
+variant type, and will instead default to `secondary` if no variant is specified.
+If `primary` is set, it will be ignored automatically.
+
+<Canvas of={ButtonStories.Sizes} />
+
+### Disabled
+
+You can disable buttons using the `disabled` property. Before doing that, think
+about whether a disabled button really makes sense for your situation — it's
+best to save this option for times when other approaches won’t quite do the job.
+
+<Canvas of={ButtonStories.Disabled} />
+
+### Full-Width
+
+Passing the `fullWidth` property will allow buttons to use the full horizontal
+space around them.
+
+<Canvas of={ButtonStories.FullWidth} />
+
+### Icons
+
+Buttons can have an icon associated with them. Pass the `iconType` property to
+the button ([a full list of icons is available here](../?path=/story/icon--list-of-icons)).
+
+You can use the `iconPosition` prop to determine whether the icon appears on
+the left or right of the text.
+
+Additionally, buttons can also be icon-only; simply pass an `iconType` and
+self-close the `Button` element:
+
+```ts
+  <Button iconType="home" aria-label="Return to the home page" />
+```
+
+If using icon-only buttons, please make sure that they are marked up correctly
+by passing `aria-label` as a prop to ensure that they are announced to
+assistive technologies correctly.
+
+<Canvas of={ButtonStories.Icons} />
+
+### Inverse
+
+Buttons can be marked as `inverse` for scenarios where they require a dark mode
+appearance. Only the `default` variant supports `inverse`; other variants will
+be ignored.
+
+<Canvas of={ButtonStories.Inverse} />
+
+### Loading
+
+Loading state can be applied to buttons by passing the `loading` prop;
+
+<Canvas of={ButtonStories.Loading} />
+
+### Wrapping Text
+
+If you require button text that does not wrap (due to e.g. limited spacing),
+use the `noWrap` prop to prevent the text from wrapping across multiple
+lines.
+
+<Canvas of={ButtonStories.WrappingText} />
+
+### HTML Button Type
+
+Standard HTML buttons can be one of three types: `button`, `submit` or
+`reset`. This functionality is available via the `type` prop, which
+allows `Button` instances to be used as standard buttons in elements
+such as forms.
+
+<Canvas of={ButtonStories.HTMLButtonType} />
+
+### As Links
```

Comment:
Question: apologies I'm likely out of the loop here but we've confirmed this is still needed? Asking as we here constantly that it's bad practice etc